packages/api/test/tx.test.ts

```diff
@@ -94,15 +96,18 @@ describe("Transaction router", async () => {
   });
 
   describe("getCount", () => {
-    it("should return the overall total transactions stat when no filters are provided", async () => {
+    it("should return the correct count when no filters are provided", async () => {
       await ctx.prisma.transactionOverallStats.populate();
+      await ctx.prisma.transactionDailyStats.populate();
 
       const { totalTransactions } = await caller.tx.getCount({});
 
       expect(totalTransactions).toBe(fixtures.canonicalTxs.length);
     });
 
     runFilterTests(async (filters) => {
+      await ctx.prisma.transactionOverallStats.populate();
+      await ctx.prisma.transactionDailyStats.populate();
```

Comment:
should we move these two lines to beforeAll()?

Reply:
These were removed due to latest tests updates